EssentialsDiscord/src/main/java/net/essentialsx/discord/JDADiscordService.java

```diff
@@ -172,6 +184,17 @@ public void startup() throws LoginException, InterruptedException {
         updateTypesRelay();
 
         Bukkit.getPluginManager().registerEvents(new BukkitListener(this), plugin);
+
+        try {
+            if (VersionUtil.getServerBukkitVersion().isHigherThan(VersionUtil.v1_12_2_R01)) {
```

Comment:
Advancements are 1.12, not 1.12.2. (There's not any reason for people to be on 1.12/1.12.1 but still better to use the correct versions if we're not doing reflective feature detection any more)